File: ML/ambuj-1211_AmbujKulshreshtha_2024csit1160_2/GettingStarted/printallprimestilln.java
import java.util.*;
public class printallprimestilln{
    public static void main(String[] args){
    int low,high;
    Scanner sc=new Scanner(System.in);
    low=sc.nextInt();
    high=sc.nextInt();
    
    for(int i=low;i<=high;i++){
        int c=0;
        for(int j=1;j<=i;j++){
            if(i%j==0){
                c++;
            }
        }
        if(c==2){
            System.out.println(i);
        }
    }
    sc.close();
    }
}

File: Android/sahilgithub908_SahilSaxena_2024IT1169_2/GettingStarted/PrintAllPrimesTillN.cpp
#include <iostream>
using namespace std;

int main(){

    int low, high,i,j;
    cin >> low >> high;
    
for(i=low;i<=high;i++)
{

j=2;
int flag=0;
    while(j*j<=i)
    {
    
if(i%j==0)
{
    flag++;
    break;
}
j++;

    }
    

if(flag==0)

    cout << i  << "\n";
        
}

return 0;

}
